docusaurus.config.js

```diff
@@ -22,13 +22,18 @@ module.exports = {
     [
       "@graphql-markdown/docusaurus",
       {
-        schema: "http://localhost:8000/graphql/",
+        schema: "./schema.graphql",
```

Comment:
@2can I think that you should also update the readme:
![image](https://user-images.githubusercontent.com/23282005/228479342-97b912fe-5473-413c-acf2-54699579f2cb.png)

Reply:
@korycins yes, this will be clarified with changes in the release of core which should provide the schema